holoviews/core/spaces.py

```diff
@@ -654,6 +687,7 @@ def _validate_key(self, key):
         Make sure the supplied key values are within the bounds
         specified by the corresponding dimension range and soft_range.
         """
+        if key == () and len(self.kdims) == 0: return ()
```

Comment:
No key and no kdims means there is nothing to do. Had to skip ``util.wrap_tuple`` as it wraps the empty tuple in another tuple.